database/migrations/2021_05_21_061857_create_questions_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateQuestionsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('questions', function (Blueprint $table) {
+            $table->id();
+            $table->integer('lesson_id');
+            $table->integer('topic_id');
+            $table->text('question');
+            $table->text('option_1');
+            $table->text('option_2');
+            $table->text('option_3');
```

Comment:
fix cứng số lượng đáp à

Reply:
dạ, em tính như vậy.
khi bắt đầu bài học em sẽ cho khoảng 20  từ để học.
học xong 1 từ thì sẽ có câu hỏi liên quan cho từ đó luôn, cho người dùng chọn đáp án và thông báo đáp án cho user,
rồi tiếp tục qua từ mới.
thì em thấy 1 câu hỏi thì nên cố định các đáp án, còn đảo lộn vị trí câu trả lời thì mình xử lý dưới client.
hay còn cách nào hay hơn anh ạ

Reply:
ví dụ:
học từ: "Programmer" bao gồm phiên âm, dịch nghĩa, tích hợp và text to speech java script
khi user học xong từ này ấn tiếp tục mình sẽ ẩn đi từ vựng đó và cho câu hỏi
câu hỏi : Từ nào sau đây được phiên dịch thành "Lập trình viên"?
a. Progamer
b. Programer
c. Programmer

Ý tưởng của em là như vậy

Reply:
ready

Reply:
làm như này không hay lắm. nên làm sao cho người ta config được số lượng đáp án, và số lượng câu trả lời đúng.
Tạm thời a merge để e làm tiếp, hãy nghĩ cách improve